analyzers/src/SonarAnalyzer.VisualBasic/Rules/ProvideDeserializationMethodsForOptionalFields.cs

```diff
@@ -37,19 +36,7 @@ public sealed class ProvideDeserializationMethodsForOptionalFields : ProvideDese
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
 
-        protected override Location GetNamedTypeIdentifierLocation(SyntaxNode node)
```

Comment:
Same thing here, it will be automatically improved. Otherwise, "rule" casing is to be fixed.